exporter/clickhouseexporter/exporter_metrics_test.go

```diff
@@ -97,6 +99,30 @@ func TestExporter_pushMetricsData(t *testing.T) {
 
 		require.Equal(t, int32(15), items.Load())
 	})
+	t.Run("check traceID and spanID", func(t *testing.T) {
+		initClickhouseTestServer(t, func(query string, values []driver.Value) error {
+			var sumMetrics = regexp.MustCompile(`^INSERT INTO otel_metrics_sum `)
+			if strings.HasPrefix(query, "INSERT INTO otel_metrics_gauge") {
+				require.Equal(t, clickhouse.ArraySet{"0102030000000000"}, values[18])
+				require.Equal(t, clickhouse.ArraySet{"01020300000000000000000000000000"}, values[19])
+			}
+			if strings.HasPrefix(query, "INSERT INTO otel_metrics_histogram") {
+				require.Equal(t, clickhouse.ArraySet{"0102030000000000"}, values[20])
+				require.Equal(t, clickhouse.ArraySet{"01020300000000000000000000000000"}, values[21])
+			}
+			if sumMetrics.MatchString(query) {
```

Comment:
Why not using HasPrefix here?

Reply:
I had tried HasPrefix but it also will match `otel_metrics_summary`

Reply:
It is hard to believe that `strings.HasPrefix(query, "INSERT INTO otel_metrics_sum ")` matches the summary. Did you by mistake remove the space after `m`?